style.css

```diff
@@ -0,0 +1,364 @@
+  -webkit-transform: translateY(0);
+          transform: translateY(0);
+}
+
+.section-one-nav__list li:first-child:active,
+.section-one-nav li:first-child:hover,
+.section-one-nav li:last-child:active,
+.section-one-nav li:last-child:hover {
+  -webkit-box-shadow: 0 0 0 transparent;
+          box-shadow: 0 0 0 transparent;
+}
+
+.section-one-nav__list,
+.section-one-nav img {
+  position: relative;
+  top: 10px;
+}
+
+.section-one-nav__twitter-account-holder {
+  display: -ms-grid;
+  display: grid;
+  -ms-grid-rows: 6rem auto;
+      grid-template-rows: 6rem auto;
+  margin: 0 2rem 0 0;
+  border-radius: 16px;
+  padding: 5px;
+}
+
+@media only screen and (max-width: 1282px) {
+  .section-one-nav__twitter-account-holder {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-rows: 5rem 6rem;
+        grid-template-rows: 5rem 6rem;
+    place-items: center;
+    margin: 0;
+  }
+}
+
+.section-one-nav__twitter-account-holder:hover {
+  background-color: #636161;
+  border: 2px sold black;
+  color: white;
+}
+
+.section-one-nav__acount-contact,
+.section-one-nav span {
+  display: block;
+  padding: 0.2rem;
+  position: relative;
+  top: -25px;
+}
+
+.section-one-nav__account, .section-one-nav__contact {
+  font-size: 0.8rem;
+}
+
+@media only screen and (max-width: 1282px) {
+  .section-one-nav__account, .section-one-nav__contact {
+    text-align: center;
+    font-size: 0.9rem;
+  }
+}
+
+.section-one-nav__contact {
+  color: white;
+  background-color: black;
+  border-radius: 16px;
+}
+
+@media only screen and (max-width: 1282px) {
+  .section-one-nav__profile-image {
+    margin-right: 5px;
+  }
+}
+
+.section-one-main {
+  padding: 2rem 1.8rem;
+  border-left: 2px solid #1da0f2;
+  border-right: 2px solid #1da0f2;
+}
+
+.section-one-main__heading {
+  padding: 0;
+  border-bottom: 2px solid #1da0f2;
+}
+
+.section-one-main__input {
+  height: 3rem;
+  border-radius: 2rem;
+}
+
+.section-one-main__list-button {
+  display: -ms-grid;
+  display: grid;
+  -ms-grid-columns: auto 1fr;
+      grid-template-columns: auto 1fr;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+}
+
+.section-one-main ul {
+  list-style: none;
+  display: -ms-grid;
+  display: grid;
+  -ms-grid-columns: 2rem 2rem 2rem 2rem 2rem 10rem;
+      grid-template-columns: 2rem 2rem 2rem 2rem 2rem 10rem;
+}
+
+.section-one-main li:last-of-type {
+  display: -ms-grid;
+  display: grid;
+  -webkit-box-pack: end;
+      -ms-flex-pack: end;
+          justify-content: end;
+}
+
+.section-one-main__tweet {
+  display: -ms-grid;
+  display: grid;
+  grid-auto-flow: row;
+}
+
+.section-one-main__tweet-profile-message {
+  display: -ms-grid;
+  display: grid;
+  -ms-grid-columns: 4rem 18rem;
+      grid-template-columns: 4rem 18rem;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  border-bottom: 2px solid #1da0f2;
+}
+
+.individual-tweet {
```

Comment:
I like the way you name your classes, very descriptive of what they are targeting.